keyboards/nckiibs/vaguettelite/vaguetteLite.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2020 noclew
```

Comment:
This file needs to be renamed to lowercase.

Reply:
@fauxpark Changed. Thanks for pointing it out.